mesh_segmenter/test/utest.cpp

```diff
@@ -17,60 +17,63 @@
 #include <gtest/gtest.h>
 #include <mesh_segmenter/mesh_segmenter.h>
 
+// This whole test has been commented 12/31/2018 1) because it doesn't work with Travis and 2) Because it depends on vtk_viewer which is unnecessary for this library.
+// It should/will be replaced by something with Gtest pass/fail criteria.
+
 // This test displays the output of the mesh segmenter as applied to a cube.  Result should be
 // a cube with each face colored a different color.
 
 TEST(ViewerTest, TestCase1)
 {
-  // generate object
-  vtkSmartPointer<vtkCubeSource> cubeSource = vtkSmartPointer<vtkCubeSource>::New();
-  cubeSource->SetBounds(-2.0, 2.0, -2.0, 2.0, -2.0, 2.0);
-  cubeSource->Update();
+//  // generate object
+//  vtkSmartPointer<vtkCubeSource> cubeSource = vtkSmartPointer<vtkCubeSource>::New();
```

Comment:
This is in test code, so I'm not going to complain about commented out code. However, you might consider removing it.

Reply:
I left it because I am not replacing it with anything better in this PR. I figured I should just leave the code commented, so someone could uncomment it, add the dependency, and run it if they wanted to in the meantime.